project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/experiment/measurement/MeasurementLookupImplementation.java

```diff
@@ -111,6 +109,7 @@ private Specification<ProteomicsMeasurement> generateProteomicsFilterSpecificati
             ontologyNameContains,
             ontologyLabelContains,
             facilityContains,
+            fractionContains,
```

Comment:
why was this changed in this PR and what does it mean?

Reply:
This is a specification that is responsible for the search within the fraction name of a proteomics measurement. It was commented out unnecessarily since the user might want to search for this property as well 🤔